### DISCUSSION TITLE
Question about upgrading v3 to v4

### DISCUSSION THREAD

**Discussion by daveajones** [NaturalIntelligence/fast-xml-parser#444] [Q&A]
I'm about to upgrade from v3 to v4 on a large project and wondered what the breakage might be.  It looks like the breaking changes are mostly with configuration property naming.  Is there anything else big to look out for?

**Comment by amitguptagwl** [comment]
[Changelog](https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/CHANGELOG.md) can help you to understand the changes in both version. You can create some sample program with version 3 and 4 for better understanding. If you stuck somewhere then let me know.
